beta-src/src/components/ui/WDInfoDisplay.tsx

```diff
@@ -0,0 +1,99 @@
+import * as React from "react";
+import {
+  Box,
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  TableContainer,
+} from "@mui/material";
+import GameMode from "../../enums/GameMode";
+import Season from "../../enums/Season";
+import Ranking from "../../enums/Ranking";
+import IntegerRange from "../../types/IntegerRange";
+
+/**
+ * game setting datas which would be passed to the component by parent component/ context/redux store
+ */
+
+interface WDInfoDisplayProps {
+  gameMode: GameMode;
+  potNumber: IntegerRange<35, 665>;
+  rank: Ranking;
+  season: Season;
+  title: string;
+  year: number;
+}
+
+const tableCellStyles = {
+  border: "none",
+  fontSize: "0.7rem",
+  p: "0 5px 0 0",
+};
+
+const WDInfoDisplay: React.FC<WDInfoDisplayProps> = function ({
+  gameMode,
+  potNumber,
+  rank,
+  season,
+  title,
+  year,
+}) {
+  return (
+    <TableContainer>
+      <Table
+        aria-label="a dense table"
+        size="small"
+        sx={{
+          maxWidth: 250,
+        }}
+      >
+        <TableHead>
+          <TableRow>
+            <TableCell
+              sx={{
+                border: "none",
+                fontWeight: 600,
+                p: "2px 5px 5px 0",
+              }}
+            >
+              <Box
+                /**
+                 * by applying CSS properties, the title of the game would be able to wrap until the second line
+                 * for the second line display '...' to show it being cut-off
+                 */
+                sx={{
+                  display: "-webkit-box",
+                  lineHeight: "1.2rem",
+                  maxHeight: "2.4rem",
+                  overflow: "hidden",
+                  textOverflow: "ellipsis",
+                  wordWrap: "break-word",
+                  WebkitLineClamp: 2,
+                  WebkitBoxOrient: "vertical",
+                }}
+              >
+                {title}
+              </Box>
```

Comment:
this Box with lineclamp would be a good separate component, just saying. we could give it props to how many lines to clamp. you don't have to do this, but it would be cool for other places that might use line clamp.

Reply:
I separate it out with the name WDContentDisplay.tsx